diff --git a/node_modules/@react-native-picker/picker/android/build.gradle b/node_modules/@react-native-picker/picker/android/build.gradle
index 155b3a8..e7a4215 100644
--- a/node_modules/@react-native-picker/picker/android/build.gradle
+++ b/node_modules/@react-native-picker/picker/android/build.gradle
@@ -119,3 +119,4 @@ dependencies {
         implementation 'com.facebook.react:react-native:+'
     }
 }
+
git add patches/@react-native-picker+picker+2.9.0.patch
git commit -m "Patch for @react-native-picker/picker to fix build.gradle issue"

